casadm/cas_main.c

```diff
@@ -70,7 +70,7 @@ static struct command_args command_args_values = {
 		.stats_filters = STATS_FILTER_DEFAULT,
 		.output_format = OUTPUT_FORMAT_DEFAULT,
 		.io_class_id = OCF_IO_CLASS_INVALID,
-		.line_size = ocf_cache_line_size_default,
+		.line_size = ocf_cache_line_size_none, //ocf_cache_line_size_default,
```

Comment:
Is there any purpose for leaving commented default cache line size?

Reply:
comments are already removed :)